jobs/run_job.sh

```diff
@@ -9,10 +9,14 @@ echo "Init job finished with exit code $result_init"
 
 if [ $result_init -eq 0 ]; then
     jen_date=$(date +%d/%m/%Y-%H:%M)
-    /usr/local/src/hyperv-networking-ci/jobs/run_tests.sh 2>&1
-    result_tempest=$?
-    echo "$ZUUL_PROJECT;$ZUUL_BRANCH;$jen_date;$ZUUL_CHANGE;$ZUUL_PATCHSET;run;$result_tempest" >> /home/jenkins-slave/networking-hyperv-statistics.log
-    echo "Tempest job finished with exit code $result_tempest"
+    if [ $RUN_TESTS == "yes" ]; then
```

Comment:
if variable is defined and is not value "X" then skip running tests, else run them. This way we make sure that the default behavior is to run the tests (what we need).